### ISSUE TITLE
test: replication/gh-3055-election-promote.test.lua flaky hangs on box.ctl.promote on OSX and FreeBSD

### ISSUE THREAD

**Issue by avtikhon** [tarantool/tarantool-qa#127]
| Tarantool 2.9.0-82-g5b6c0fdef
 | Target: Darwin-x86_64-RelWithDebInfo
 | Build options: cmake . -DCMAKE_INSTALL_PREFIX=/usr/local -DENABLE_BACKTRACE=ON
 | Compiler: /Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc /Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
 | C_FLAGS: -Wno-unknown-pragmas -fexceptions -funwind-tables -fno-omit-frame-pointer -fno-stack-protector -fno-common -msse2 -std=c11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-gnu-alignof-expression -Werror
 | CXX_FLAGS: -Wno-unknown-pragmas -fexceptions -funwind-tables -fno-omit-frame-pointer -fno-stack-protector -fno-common -msse2 -std=c++11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-invalid-offsetof -Wno-gnu-alignof-expression -Werror

Issue:
```
[008] replication/gh-3055-election-promote.test.lua                   Test timeout of 310 secs reached	[ fail ]
[008] 
[008] Test failed! Result content mismatch:
[008] --- replication/gh-3055-election-promote.result	Sat Jun 12 19:12:44 2021
[008] +++ /tmp/tnt/rejects/replication/gh-3055-election-promote.reject	Sat Jun 12 19:32:38 2021
[008] @@ -44,62 +44,3 @@
[008]   | ---
[008]   | ...
[008]  box.ctl.promote()
[008] - | ---
[008] - | ...
[008] -assert(box.info.election.state == 'leader')
```

**Comment by avtikhon** [comment]
Seems that the issue will be fixed within tarantool/tarantool#5430 :
https://github.com/tarantool/tarantool/issues/5430#issuecomment-864550359

**Comment by NickVolynkin** [comment]
Doesn't reproduce now. Most probably, it was fixed in https://github.com/tarantool/tarantool/issues/5430
